src/_P073_7DGT.ino

```diff
@@ -404,32 +413,33 @@ void LogBufferContent(String prefix) {
     }
   }
 
-  int     dotpos;
-  uint8_t showbuffer[8];
-  bool    showperiods[8];
-  byte    spidata[2];
-  uint8_t pin1, pin2, pin3;
-  byte    displayModel;
-  byte    output;
-  byte    brightness;
-  bool    timesep;
-  bool    shift;
-  bool    periods;
-  bool    hideDegree;
-  bool    rightAlignTempMAX7219;
-  uint8_t fontset;
+  int      dotpos;
+  uint8_t  showbuffer[8];
+  bool     showperiods[8];
+  byte     spidata[2];
+  uint8_t  pin1, pin2, pin3;
+  byte     displayModel;
+  byte     output;
+  byte     brightness;
+  bool     timesep;
+  bool     shift;
+  bool     periods;
+  bool     hideDegree;
+  bool     rightAlignTempMAX7219;
+  uint8_t  fontset;
 #ifdef P073_7DBIN_COMMAND
-  bool    binaryData;
+  bool     binaryData;
 #endif // P073_7DBIN_COMMAND
 #ifdef P073_SCROLL_TEXT
-  bool    txtScrolling;
-  uint8_t scrollCount;
-  uint8_t scrollPos;
+  bool     txtScrolling;
+  uint16_t scrollCount;
+  uint16_t scrollPos;
+  bool     scrollFull;
 private:
-  uint8_t _scrollSpeed;
-#endif
+  uint8_t  _scrollSpeed;
+#endif // P073_SCROLL_TEXT
 #if defined(P073_SCROLL_TEXT) || defined(P073_7DBIN_COMMAND)
-  String  _textToScroll;
+  String   _textToScroll;
```

Comment:
Have not checked them all, but this list does seem longer than the ones initialized in the constructor.
Maybe better to make sure to initialize them here when declaring so none is forgotten.